src/models/searchservice/request/VerticalSearchRequest.ts

```diff
@@ -39,7 +38,7 @@ export interface VerticalSearchRequest extends SearchRequest {
   /** Enables session tracking. */
   sessionTrackingEnabled?: boolean,
   /** The static filters to apply to the search. */
-  staticFilters?: CombinedFilter | Filter,
+  staticFilters?: StaticFilter,
```

Comment:
This feels a bit odd, the attribute name is plural, but we only allow a singular value.

Reply:
I agree, this looks a little weird. Although, this was the case before also where we accepted a single value (either a `Filter` or a `CombinedFilter`) for the plural `staticFilters`. It was just slightly obscured because of the union type.
Would it make sense to rename the attribute to `staticFilter`? Because that's what the user is giving us; it's just that they can combine a bunch of field value filters together if they want to in their one static filter

Reply:
Yeah, renaming to static filter sounds good!